test/c3/appframework/appframework_test.go

```diff
@@ -3049,4 +3049,73 @@ var _ = Describe("c3appfw test", func() {
 			testenv.VerifyRFSFMet(ctx, deployment, testcaseEnvInst)
 		})
 	})
+
+	Context("Clustered deployment (C3 - clustered indexer, search head cluster)", func() {
+		It("integration, c3: can deploy a C3 SVA and a Standalone, then add that Standalone as a Search Head to the cluster", func() {
+
+			/* Test Steps
+			   ################## SETUP ###################
+			   * Deploy C3 CRD
+			   * Deploy Standalone with clusterMasterRef
+			   ############# VERIFICATION #################
+			   * Verify clusterMasterRef is present in Standalone's server.conf file
+			*/
+			//################## SETUP ####################
+			// Deploy C3 CRD
+			indexerReplicas := 3
+			testcaseEnvInst.Log.Info("Deploy Single Site Indexer Cluster")
+			err := deployment.DeploySingleSiteCluster(ctx, deployment.GetName(), indexerReplicas, false, "")
+			Expect(err).To(Succeed(), "Unable to deploy Single Site Indexer Cluste")
+
+			// Create spec with clusterMasterRef for Standalone
+			spec := enterpriseApi.StandaloneSpec{
+				CommonSplunkSpec: enterpriseApi.CommonSplunkSpec{
+					Spec: enterpriseApi.Spec{
+						ImagePullPolicy: "Always",
+					},
+					Volumes: []corev1.Volume{},
+					ClusterMasterRef: corev1.ObjectReference{
+						Name: deployment.GetName(),
+					},
+				},
+			}
+
+			// Deploy Standalone with clusterMasterRef
+			testcaseEnvInst.Log.Info("Deploy Standalone with clusterMasterRef")
+			standalone, err := deployment.DeployStandaloneWithGivenSpec(ctx, deployment.GetName(), spec)
```

Comment:
The point of this test was to add standalone as single SH to the Cluster master. I think without adding the extra yaml string Standalone wont be configured as SH on CM. Please modify the test accordingly.